spec/requests/api/v1/events/events_spec.rb

```diff
@@ -26,3 +26,59 @@
     )
   end
 end
+
+describe 'POST/v1/events' do
+  it 'saves the address, lat, lon, name, and started_at date' do
+    date = Time.zone.now
+    device_token = '123abcd456xyz'
+    owner = create(:user, device_token: device_token)
+
+    post '/v1/events', {
```

Comment:
It occurs to me after @brimatteng did this, you can use [attributes_for](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#using-factories) in FactoryGirl to build out this data instead of manually entering it.